### PR TITLE
adding models with specs

### PR DESCRIPTION
**Pull request by nir0**

### CODE REVIEW

File: app/models/attachment.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Attachment < ApplicationRecord
```
[the remark this answers is not in the document]
Comment (rapidos) [reply app/models/attachment.rb:3]: @nir0

File: config/initializers/money.rb [hunk 2]
```diff
@@ -71,7 +71,7 @@
   #
   # set to BigDecimal::ROUND_HALF_EVEN by default
   #
-  # config.rounding_mode = BigDecimal::ROUND_HALF_UP
+  config.rounding_mode = BigDecimal::ROUND_HALF_UP
```
Comment (rapidos) [inline config/initializers/money.rb:74]: @nir0 why? :)

**Review by rapidos — COMMENTED** [review]
@nir0 you are missing class comments.

Also when commenting, please, always think about answering those 2 questions: **what** and **why**

File: config/initializers/money.rb (hunk 2 above)
Comment (nir0) [reply config/initializers/money.rb:74]: to remove deprecation warning

File: app/models/attachment.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (nir0) [reply app/models/attachment.rb:3]: done

File: app/models/invoice.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+
```
[the remark this answers is not in the document]
Comment (nir0) [reply app/models/invoice.rb]: done

**Comment by balbesina** [comment]
@rapidos, why don't we use activeStorage? it seems very natural here, inside the rails environment.

**Review by rapidos — APPROVED** [review]

**Pull request merged**